slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/element/container/SingleBlockElementContainer.kt

```diff
@@ -9,7 +9,7 @@ import com.slack.api.model.kotlin_extension.block.element.dsl.BlockElementDsl
 /**
  * Supports a BlockElementContainer holding exactly one block element
  */
-class SingleBlockElementContainer() : BlockElementDsl {
```

Comment:
I see this could be slightly better, but can you remove unrelated changes like this from this PR?

Reply:
Resolved in 1be897c8ce41fb2d32ee1b4cdaa0d647fd7b43c0